ghcide/exe/Arguments.hs

```diff
@@ -19,6 +20,9 @@ data Arguments = Arguments
     ,argsVerbose                    :: Bool
     ,argsCommand                    :: Command
     ,argsConservativeChangeTracking :: Bool
+#ifdef MONITORING_EKG
```

Comment:
I wonder if it would be simpler to define this argument unconditionally and simply have it do nothing if we haven't built with EKG support. This way is also fine, I just hate CPP :sweat_smile:

Reply:
It's too late now, CPP is everywhere :(